jest.config.ts

```diff
@@ -9,11 +9,16 @@ const createJestConfig = nextJest({
 // Add any custom config to be passed to Jest
 const config: Config = {
   coverageProvider: "v8",
-  testEnvironment: "jsdom",
+  testEnvironment: "node",
```

Comment:
We may need to use jsdom here to actually test the the form validation is triggered in the control as this code 
 is triggered client side.

Reply:
https://github.com/Snowfork/snowbridge-app/pull/13/commits/2c2cde9f2f217e009b448168cd075d36803855d1